shared/util/phone-numbers/index.tsx

```diff
@@ -121,4 +121,15 @@ export const formatPhoneNumber = (rawNumber: string) => {
   return `+${number.getCountryCode()} ${phoneUtil.format(number, PNF.NATIONAL)}`
 }
 
+// Return phone number in international format, e.g. +1 (800) 555 0123
+// or e.164 if parsing fails
+export const e164ToDisplay = (e164: string) => {
+  try {
+    const number = phoneUtil.parse(e164)
+    return phoneUtil.format(number, PNF.INTERNATIONAL)
```

Comment:
International format gives `+1 345-543-1232` - it would be nice if we did a special case to get the parens for US/cCA numbers with `+1 (234) 555-6789`

Reply:
Agreed. I'm working on another PR based off this one, I'll look into it in that one.